src/STW.SubmissionApi/Services/IValidationService.cs

```diff
@@ -0,0 +1,8 @@
+using NJsonSchema.Validation;
+
+namespace STW.SubmissionApi.Services;
+
+public interface IValidationService
+{
+    public Task<List<ValidationError>> Validate(string spsCertificate);
```

Comment:
Does it matter that much because it's a fully internal interface (only using an interface so it can be used in dependency injection)? If I change this to `ICollection` then I have to do `errors.ToList().ForEach(` in the controller where I use it.